modules/ROOT/pages/_partials/app-security/configuring-dev-env-app-security.adoc

```diff
@@ -0,0 +1,26 @@
+= Configuring your Development Environment for the {app-security-service} Service
+
+:service-name: App Security
+
+== Downloading the Mobile Services Configuration File
```

Comment:
@austincunningham I think we can import [this file](https://github.com/aerogear/mobile-docs/blob/master/modules/ROOT/pages/_partials/generic-obtaining-the-mobile-sdk-config-file.adoc) here.

Reply:
@finp is there any generic one of these files as we don't need all the platforms here?

Reply:
as discussed that contains more that just cordova so I didn't use it .

Reply:
I'm ok with the duplication, if that's the concern here?